randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1276,8 +1302,25 @@ Extra - room_id: [36]
       All of the following:
           Morph Ball and Screw Attack and Wave Beam
           Any of the following:
-              # Get past Rippers - TODO: health for damage runs
-              Space Jump or Damage Runs (Intermediate)
+              # Get past Rippers
+              Space Jump
+              All of the following:
+                  Damage Runs (Beginner)
+                  Any of the following:
+                      Normal Damage ≥ 50
+                      # Suitless
+                      Lava Damage ≥ 100 and Normal Damage ≥ 50
+                      All of the following:
+                          # 1 suit
+                          Lava Damage ≥ 50 and Normal Damage ≥ 30
+                          Gravity Suit or Varia Suit
+                      # Both suits
+                      Gravity Suit and Varia Suit and Lava Damage ≥ 25 and Normal Damage ≥ 15
+                      All of the following:
+                          Combat (Advanced)
+                          Any of the following:
+                              Lava Damage ≥ 25
+                              Gravity Suit and Varia Suit and Lava Damage ≥ 5
```

Comment:
❓ "Gravity or Varia" instead of and?  All three damage amounts (Varia only, Grav only, Varia+Grav) are going to be pretty close together, within about 5 points.

Reply:
It's 25dmg or (grav+varia+5dmg). I agree it's been confusing because in another part I also listed dmg for one suit, but this I only have 0 suits or both suits. I think it'd been Miepee to suggest I only bother putting these, since 1 suit is 15dmg, not a big enough difference from others to bother distinguishing.

Reply:
Yeah I'm suggesting that because there's little difference between 1 and 2 suits, offering the 1 suit path instead potentially gives the logic more options and would effectively cover the 2 suit path (just treating having both the same as having one)

Reply:
Opting against, as it potentially encourages both suits in logic to face Ridley, or is unlikely to affect other checks any more significantly. TBH I prefer all 3 options covered.